db/compaction_iterator.cc

```diff
@@ -1,5 +1,3 @@
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file. See the AUTHORS file for names of contributors.
```

Comment:
I took a closer look at the content of the file. I think they are a total rewrite (which is much much more complicated than initial implementation). No code was copied from previous LevelDB code.

Other files were brand new. Not even a rewriting of existing logic.